lib/spirv/lib/src/transpiler.dart

```diff
@@ -1,8 +1,8 @@
 // Copyright 2013 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-
```

Comment:
I think this blank line needs to be here so it's not part of the copyright header